<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="linkedTable">
     <property name="styleSheet">
      <string notr="true">;</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="WharisLinked">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; text-decoration: underline; color:#000000;&quot;&gt;What is linked&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTableWidget" name="unLinkedTable">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>&lt;&gt;</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="WhatIsNotLinked">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; text-decoration: underline; color:#000000;&quot;&gt;What is not linked&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="unlinkButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 150, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Unlink</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="linkButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 150, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Link</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
